Source/Scene/PostProcessStageLibrary.js

```diff
@@ -16,12 +17,16 @@ define([
         '../Shaders/PostProcessStages/DepthOfField',
         '../Shaders/PostProcessStages/DepthView',
         '../Shaders/PostProcessStages/EdgeDetection',
+        '../Shaders/PostProcessStages/FilmicTonemapping',
         '../Shaders/PostProcessStages/FXAA',
         '../Shaders/PostProcessStages/GaussianBlur1D',
         '../Shaders/PostProcessStages/LensFlare',
+        '../Shaders/PostProcessStages/ModifiedReinhardTonemapping',
         '../Shaders/PostProcessStages/NightVision',
+        '../shaders/PostProcessStages/ReinhardTonemapping',
```

Comment:
lowercase shaders here is breaking linux.

Reply:
Thanks. Fixed.